tests/topotests/bgp_ipv4_over_ipv6/test_rfc5549_ebgp_nbr.py

```diff
@@ -132,7 +132,7 @@ def setup_module(mod):
         pytest.skip(tgen.errors)
 
     BGP_CONVERGENCE = verify_bgp_convergence(tgen, topo)
-    assert BGP_CONVERGENCE is True, "setup_module :Failed \n Error:" " {}".format(
+    assert BGP_CONVERGENCE is True, "setup_module :Failed \n Error:{}".format(
```

Comment:
There appears to be no space between the text `Error:` and the error message; please fix this.